pyperformance/data-files/requirements.txt

```diff
@@ -2,17 +2,13 @@
 # This file is autogenerated by pip-compile with Python 3.11
 # by the following command:
 #
-#    pip-compile --output-file=requirements.txt requirements.in
+#    pip-compile --output-file=pyperformance/data-files/requirements.txt requirements.in
 #
-packaging==21.3
+packaging==23.1
     # via -r requirements.in
-psutil==5.9.0
+psutil==5.9.5
     # via
     #   -r requirements.in
     #   pyperf
-pyparsing==3.0.8
-    # via packaging
 pyperf==2.6.0
     # via -r requirements.in
-toml==0.10.2
-    # via -r requirements.in
```

Comment:
I ran `pip-compile` to regenerate this file, and it also bumped `packaging` and `psutil`. Not sure if that's desirable or if I did something wrong there!